tools/local_fix_format.sh

```diff
@@ -22,20 +22,24 @@
 # If DISPLAY is set, then tkdiff pops up for some BUILD changes.
 unset DISPLAY
 
+# The following optional argument is added to be able to run this script using Docker,
+# due to a problem to locate clang using WSL on Windows. Call with -docker as the first arument.
```

Comment:
what's WSL?

Reply:
[WSL ](https://learn.microsoft.com/en-us/windows/wsl/install) is a windows based solution to support Linux VM on a Windows host